cookbook/serializer.rst

```diff
@@ -48,19 +46,44 @@ it in your configuration:
         $container->loadFromExtension('framework', array(
             // ...
             'serializer' => array(
-                'enabled' => true
+                'enabled' => true,
             ),
         ));
 
+Using the Serializer Service
+---------------------------------
```

Comment:
this tagline, as reSt calls it, should have exact the same length as the title text.